io-ballerina/file_csv_io.bal

```diff
@@ -43,7 +43,7 @@ public isolated function fileReadCsvAsStream(@untainted string path) returns @ta
 # + path - The CSV file path
 # + content - CSV content as an array of string arrays
 # + option - To indicate whether to overwrite or append the given content
-# + return - Either an `io:Error` or the null `()` value when the writing was successful
+# + return - An `io:Error` or a nil `()` value when the writing was successful
```

Comment:
Seems "a nil `()` value" is kind of repeating. Can't we just say "a nil value"

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/179/commits/a566c8db275b7b9c56a7d25fd661f1141c4db7bd